wf-federated/Snakefile

```diff
@@ -0,0 +1,44 @@
+configfile: "/data/config.yaml"
+
+from snakemake.remote.S3 import RemoteProvider as S3RemoteProvider
+
+# any other remote storage is possible as well e.g. FTP
+# Remote storage has to be suported by Snakemake.
+# https://snakemake.readthedocs.io/en/stable/snakefiles/remote_files.html
+
+S3 = S3RemoteProvider(access_key_id=os.environ["ACCESS_KEY_ID"], 
+                      secret_access_key=os.environ["SECRET_ACCESS_KEY"], 
+                      endpoint_url="https://s3-elixir.cloud.e-infra.cz")
```

Comment:
In order to separate configuration from the descriptor file entirely, maybe we could set a config var for the URL as well?